tools/database/src/analyse.rs

```diff
@@ -0,0 +1,271 @@
+use clap::Parser;
+use near_store::db::RocksDB;
+use near_store::{col_name, DBCol};
+use rayon::prelude::*;
+use rocksdb::{ColumnFamilyDescriptor, Options, DB};
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+use std::{panic, println};
+use strum::IntoEnumIterator;
+
+#[derive(Parser)]
+pub struct AnalyseDatabaseCommand {
+    #[arg(short, long)]
+    /// If specified only yhis column will be analysed
+    column: Option<String>,
+
+    #[arg(short, long, default_value_t = 100)]
+    /// Number of count sizes to ourput
+    top_k: usize,
+}
+
+fn get_db_col(col: &str) -> Option<DBCol> {
+    for db_col in DBCol::iter() {
+        if <&str>::from(db_col) == col {
+            return Some(db_col);
+        }
+    }
+    return None;
+}
+
+fn get_all_col_family_names() -> Vec<DBCol> {
+    DBCol::iter().collect()
+}
+
+#[derive(Clone)]
+struct ColumnFamilyCountAndSize {
+    number_of_pairs: usize,
+    size: usize,
+}
+
+struct DataSizeDistributionResults {
```

Comment:
nit: I would drop `Results` suffix since it is somewhat confusing considering widely used `Result` in rust